docs/cloud/how-to-manage-certificates-in-temporal-cloud.md

```diff
@@ -0,0 +1,133 @@
+---
+id: how-to-manage-certificates-in-temporal-cloud
+title: How to manage certificates in Temporal Cloud
+sidebar_label: Certificates
+description: Access to Temporal Cloud requires a certificate from you.
+---
+
+Access to Temporal Cloud is secured with the mutual Transport Layer Security (mTLS) protocol.
+This protocol requires a certificate from you.
+
+The benefits of using a CA certificate include the following:
+
+- You can use your CA to generate client certificates to comply with your security policies for certificate expiration and rotation.
+  Client certificates can be generated and rotated without having to configure or update public certificates shared with Temporal.
+- Temporal does not need to receive certificate private keys.
+  When you configure access to Temporal Cloud, no exchange of secrets is required.
+
+## Requirements for client certificates
+
+Certificates provided to Temporal for your [Namespaces](/namespaces) _must_ meet one of the following two sets of requirements.
+
+### Option 1: End-entity certificates
+
+An end-entity certificate _must_ meet the following criteria:
+
+- The certificate must be X.509v3.
+- Basic constraints must include `CA: false`.
+- The key usage must include Digital Signature.
+- The signing algorithm must include SHA-256 or stronger.
+  SHA-1 and MD5 signing algorithms are rejected.
+
+When a client presents an end-entity certificate, and the whole certificate chain is constructed, each certificate in the chain (from end-entity to the root) must have a unique Distinguished Name (DN).
+
+### Option 2: CA bundle certificates
+
+CA bundle certificates _must_ meet the following criteria:
+
+- The certificates must be X.509v3.
+- Each certificate in the bundle must be either a root certificate or issued by another certificate in the bundle.
+- Each certificate in the bundle must include `CA: true`.
+- A certificate cannot be a well-known CA (such as DigiCert or Let's Encrypt) _unless_ the user also specifies certificate filters.
+- The signing algorithm must include SHA-256 or stronger. SHA-1 and MD5 signing algorithms are rejected.
+
+## Generate CA and client certificates
+
+Temporal Cloud authenticates a client connection by validating the client certificate against one or more CA certificates that are configured for the specified Namespace.
+
+### You already have certificate management infrastructure
```

Comment:
The `You already have` and `You have no` sections are really two different paths but presented flat like this, that isn't very clear. 
As a fast-follow iteration (not in this PR) maybe we could present them as "Options" or something to make that a little more like a decision tree.

Reply:
Agreed. This one has been nibbling at me for a while.